test/scripts/detect-fork.ts

```diff
@@ -73,6 +89,146 @@ const fetchImported = async (api: ApiPromise, hash: string) => {
     .toJSON();
 };
 
+const getContracts = (url: string): Promise<any> => {
+  return new Promise<any>(function (ok, err) {
+    const buffers: Array<Buffer> = [];
+    https
+      .get(url, (res) => {
+        if (res.statusCode === 200) {
+          res.on("data", (d) => {
+            buffers.push(d);
+          });
+          res.on("end", () => {
+            ok(JSON.parse(Buffer.concat(buffers).toString("utf8")));
+          });
+        } else err(res.statusMessage);
+      })
+      .on("error", (e) => {
+        err(e);
+      });
+  });
+};
+
+const NONCES = [
+  {
+    name: "BasicInboundChannel",
+    event: "MessageDispatched",
+    storageKey:
+      "0x684b82bef882079feeabe54a5bd7b94a718368a0ace36e2b1b8b6dbd7f8093c0",
+  },
+  {
+    name: "IncentivizedInboundChannel",
+    event: "MessageDispatched",
+    storageKey:
+      "0xf0f4d0b91e760c07da58bc0498033acb718368a0ace36e2b1b8b6dbd7f8093c0",
+  },
+  {
+    name: "BasicOutboundChannel",
```

Comment:
Can remove everything related to parachain outbound channels. The parachain outbound channels won't be affected in any way by a hard fork on ropsten. The same applies to the inbound channels on the ethereum side that receive messages from the parachain outbound channels.

Reply:
I left the config for these nonces for displaying nonces in the script output. However I have changed the `generateUpdates` function to only generate updates for parachain inbound channels.